### PR TITLE
allow both approvers and assignees in owners files

### PR DESCRIPTION
**Pull request by grodrigues3**
In preparation for the upcoming owners change, let's allow both assignees (old way) and approvers (new way) to lgtm code.  This does not affect the lgtm functionality.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1888)

### CODE REVIEW

File: mungegithub/features/repo-updates.go
Comment (foxish) [inline mungegithub/features/repo-updates.go]: For now maybe we should merge the two sets together? That would ensure that things continue to work as we transition and afterwards, we can separate them out again.

Comment (foxish) [inline mungegithub/features/repo-updates.go]: Also, aren't we supposed to rename assignees to approvers, as they mean the same thing?

Comment (eparis) [inline mungegithub/features/repo-updates.go]: They are merged together. Although under the "old" name `o.assignees`.  This patch LGTM, it is backwards compatible...

**Review by eparis — APPROVED** [review]

**Comment by eparis** [comment]
although I would like to see a test...

**Comment by grodrigues3** [comment]
suggestions for how to test?  It doesn't seem like something that I can write a unit test for easily

**Comment by eparis** [comment]
/lgtm
uggh, we don't have any blunderbuss unit tests, do we.

**Comment by grodrigues3** [comment]
@eparis  I think /lgtm works if it's the **only** text in the comment, to avoid cases like "/lgtm after you fix..."

**Comment by foxish** [comment]
@eparis @grodrigues3 /lgtm //some comment here (this works)

**Pull request merged**
